### ISSUE TITLE
autoComplete don't clear completely when minQueryLength > 0

### ISSUE THREAD

**Issue by schlebe** [primefaces/primefaces#1159]
I have set minQueryLength = "1" to prohibit suggestions's panel when widget is cleared.

Now, when I delete the last character in autoComplete using DELETE or BACKSPACE key, the widget is cleared on screen, but on server, the value of variable linked to this widget contains the last character !

I have traced my program and I have seen that the setTireLineName() method associated to my widget is not called when I delete the last character !

I have replaced "1" by "2" in minQueryLength atttribute and I see that now, the setTireLineName() method is not called when I delete the first character of a string of 2 characters.

I think that minQueryLength prohibit widget to update value on server ... but this is not it role ! 

This has been discussed on forum in [(http://forum.primefaces.org/viewtopic.php?f=3&t=44473&p=139220#p139220)]

p:autoComplete
    id="SearchTireLineName"
    size="25"
    immediate="true"
    value="#{postLabListViewController.postLabellingSearchCriteria.tireLineName}"
    completeMethod="#{postLabListViewController.searchFormTireLineNameAutoCompleteItems}"
    minQueryLength="1"

**Comment by tandraschko** [comment]
It's hard to follow whats wrong here.
Could you please create a new issue, if still replicatable and important for you, with all questsions filled in our new issue template?
